README.md

```diff
@@ -245,10 +245,13 @@ Please see the <a href="http://docs.hazelcast.org/docs/latest/manual/html-single
 <property name="hibernate.cache.hazelcast.use_native_client">true</property>
 ```
 
-To set up Native Client, add the Hazelcast **group-name**, **group-password** and **cluster member address** properties. Native Client will connect to the defined member and will get the addresses of all members in the cluster. If the connected member dies or leaves the cluster, the client will automatically switch to another member in the cluster.
+To set up Native Client, add the Hazelcast **group-name**, **group-password** and **cluster member address** (you can
```

Comment:
I think this is outdated, group name and group password were replaced by cluster name in 4.x

Reply:
Fixed: https://github.com/hazelcast/hazelcast-hibernate/pull/385/commits/32477abe3dfd5c24563de455571020bed260a60c